src/client/components/informationPanel/InformationCard.js

```diff
@@ -29,210 +28,46 @@ const InformationCard = () => {
-      <IconLocation width="13px" height="13px" color={theme.textTertiary} />
-      {location ? (
-        <Text size="normal" color={theme.textTertiary}>
-          {location}
-        </Text>
-      ) : (
-        <Text
-          css={`
-            cursor: pointer;
-          `}
-          color={theme.accent}
-          onClick={() => dispatchModal(open('basicInformation'))}
-        >
-          Add location
-        </Text>
-      )}
-    </Social>
-  )
-
-  RenderLocation.propTypes = { location: PropTypes.string }
-
-  const RenderEmpty = () => (
-    <Center height="130px">
-      <Text.Block
-        css={`
-          text-align: center;
-        `}
-        size="xlarge"
-      >
-        You have no name, bio or location
-      </Text.Block>
-      <Text
-        css={`
-          cursor: pointer;
-        `}
-        size="small"
-        color={theme.accent}
-        onClick={() => dispatchModal(open('basicInformation'))}
-      >
-        Add basic information
-      </Text>
-    </Center>
-  )
-
-  const RenderGitHub = ({ github: { username } }) => (
-    <Social>
-      <IconGitHub width="13px" height="13px" color={theme.textTertiary} />
-      {username ? (
-        <div>
-          <Link href={`https://github.com/${username}`}>{username}</Link>
-          <Verified />
-        </div>
-      ) : (
-        <div>
-          <Button
-            compact
-            mode="outline"
-            css={`
-              position: relative;
-            `}
-            onClick={() => setPopover('github')}
-          >
-            Verify my GitHub account
-          </Button>
-          {activePopover === 'github' && <PopoverCard social="GitHub" />}
-        </div>
-      )}
-    </Social>
-  )
-
-  RenderGitHub.propTypes = { github: PropTypes.object }
-
-  const RenderTwitter = ({ twitter: { username } }) => (
-    <Social>
-      <IconTwitter width="13px" height="13px" color={theme.textTertiary} />
-      {username ? (
-        <div>
-          <Link href={`https://twitter.com/${username}`}>{username}</Link>
-          <Verified />
-        </div>
-      ) : (
-        <div>
-          <Button
-            compact
-            mode="outline"
-            css={`
-              position: relative;
-            `}
-            onClick={() => setPopover('twitter')}
-          >
-            Verify my Twitter account
-          </Button>
-
-          {activePopover === 'twitter' && <PopoverCard social="Twitter" />}
-        </div>
-      )}
-    </Social>
-  )
-
-  RenderTwitter.propTypes = { twitter: PropTypes.object }
-
-  const RenderWebsite = ({ website }) =>
-    website && (
-      <Social>
-        <IconGlobe width="13px" height="13px" color={theme.textPrimary} />
-        <Link href={website} placeholder="website" size="small">
-          {website}
-        </Link>
-      </Social>
-    )
-
-  RenderWebsite.propTypes = { website: PropTypes.string }
+  const handleOpenEdit = () => dispatchModal(open('basicInformation'))
```

Comment:
Basic Info Panel got split into smaller modules, but modules opening basicInformation Modal are only like helpers, small and totally dependent on their parent. That's why using props instead of making subcomponents more independent seemed good enough - I'd leave it as it is, if it's ok? I don't mind changing it of course :)

Reply:
Yes, that's fine.